README.md

```diff
@@ -18,24 +18,31 @@ Install the `RectorLaravel` package as dependency:
 composer require driftingly/rector-laravel --dev
 ```
 
-## Use Sets
+## Automate Laravel Upgrades
 
-To add a set to your config, use `RectorLaravel\Set\LaravelSetList` class and pick one of the constants:
+To add a set to your config, use `RectorLaravel\Set\LaravelLevelSetList` and pick the constant that matches your target version.
+Sets for higher versions include sets for lower versions.
 
 ```php
-<?php
+return Rector\Config\RectorConfig::configure()
+    ->withSets([
+        RectorLaravel\Set\LaravelLevelSetList::LARAVEL_110,
+    ]);
+```
 
-declare(strict_types=1);
+## Additional Sets
 
-use Rector\Config\RectorConfig;
-use RectorLaravel\Set\LaravelSetList;
+The other sets do not correlate directly to Laravel upgrades, but can be used to improve different aspects of your code.
```

Comment:
This is not necessarily true, since sets like `LaravelSetList::LARAVEL_110` can directly help with upgrades, just without inheriting all the previous version changes. Maybe we can point that out?

Reply:
I changed this to be more accurate.